src/policy/fees.cpp

```diff
@@ -12,6 +12,7 @@
 #include <policy/feerate.h>
 #include <primitives/transaction.h>
 #include <random.h>
+#include <scheduler.h>
```

Comment:
I think it's a bit strange for policy/fees to have a dependency on scheduler. Maybe register `FlushFeeEstimates()` with the scheduler within `AppInitMain()` instead?

Reply:
Sure :+1: , fixed.